config/env.go

```diff
@@ -29,6 +29,12 @@ const (
 	envVarSCADAAddress = "HCP_SCADA_ADDRESS"
 
 	envVarSCADATLS = "HCP_SCADA_TLS"
+
+	envVarHCPOrganizationID = "HCP_ORGANIZATION_ID"
+
+	envVarHCPProjectID = "HCP_PROJECT_ID"
+
+	envVarHCPEnvironment = "HCP_ENVIRONMENT"
```

Comment:
After some discussion today, let's remove the HCP environment setting from the profile.